### ISSUE TITLE
Starting stats for character 3 are missing

### ISSUE THREAD

**Issue by MarcoBrieden** [might-and-magic/mm678-guide#1]
The guide has only the character creation stats for 4 of the 5 characters: 1, 2, 4 and 5.

**Comment by tomchen** [comment]
Yeah that's true. The original article in Chinese also misses that image.  http://tieba.baidu.com/p/6354402129 I've asked the author (he's one of mmmerge community version's contributors). But I doubt if he can retrieve the image...

**Comment by MarcoBrieden** [comment]
Thank you!
